index.bs

```diff
@@ -2416,6 +2427,47 @@ This is because the first 37 bytes of the signed data in a FIDO U2F authenticati
 the same procedure as other [=assertion signatures=] generated by the [=authenticatorMakeCredential=] operation.
 
 
+## Authenticator taxonomy ## {#sctn-authenticator-taxonomy}
+
+[=Authenticator=] types vary along several different dimensions: [=attachment modality=], employed [[#transport|transport(s)]], whether they
+are [=user verification=]-capable or only [=test of user presence|detect any user's presence=], and whether they support
+[=client-side-resident credential private keys=] or not.
+
+Issue: Expand this section with further discussion about the other dimensions mentioned above.
+
+
+### <dfn>Attachment Modality</dfn> ### {#sctn-authenticator-attachment-modality}
```

Comment:
I'm inclined to fully-qualify this term, especially in the heading here, as "Authenticator Attachment Modality".

Reply:
Fixed in #979.